packages/core/admin/admin/src/content-manager/components/DynamicZone/index.js

```diff
@@ -198,26 +118,21 @@ const DynamicZone = ({
             required={fieldSchema.required || false}
           />
           {dynamicDisplayedComponents.map((componentUid, index) => {
-            const showDownIcon =
-              isFieldAllowed &&
-              dynamicDisplayedComponentsLength > 0 &&
-              index < dynamicDisplayedComponentsLength - 1;
-            const showUpIcon = isFieldAllowed && dynamicDisplayedComponentsLength > 0 && index > 0;
-            const isOpen = componentCollapses[index]?.isOpen || false;
+            const showDownIcon = isFieldAllowed && index < dynamicDisplayedComponentsLength - 1;
+            const showUpIcon = isFieldAllowed && index > 0;
 
             return (
-              <Component
+              <DynamicZoneComponent
                 componentUid={componentUid}
                 formErrors={formErrors}
+                // eslint-disable-next-line react/no-array-index-key
```

Comment:
No you wouldn't be able to because it's unique to the component which there could be X of they're not field UIDs. I did think about solving it but couldn't see an easy enough solution.